metabase/metabase
# Mantine number input fix

Closes https://github.com/metabase/metabase/issues/35312

How to verify:
- `yarn storybook`
- Open http://localhost:6006/?path=/story/inputs-numberinput--default-story
- Make sure you can enter numbers with arbitrary precision

<img width="752" alt="Screenshot 2023-11-02 at 16 04 35" src="https://github.com/metabase/metabase/assets/8542534/a0a5b2a9-9b5a-4724-9c64-56eaf98d91e9">
